integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml

```diff
@@ -0,0 +1,54 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: splunk
+  name: splunk-metrics-and-events
+spec:
+  class: supported
+  provider: events
+  display_name: Splunk Events
+  short_description: >-
+    A sensu event handler pipeline that sends events to be indexed
+    in Splunk via a Universal Forwarder
+  supported_platforms:
+    - darwin
+    - linux
+    - windows
+  tags:
+    - splunk
+    - events
+  contributors:
+    - "@sensu"
+    - "@nixwiz"
+  prompts:
+    - type: section
+      title: Splunk Endpoint
+    - type: markdown
+      body: |
+        This integration requires a TCP endpoint to send sensu events to
+    - type: question
+      name: host
+      input:
+        type: string
+        title: Host Name
+        default: localhost
+    - type: question
+      name: port
+      input:
+        type: integer
+        title: Port
+        default: 7758
+  resource_patches:
+    - resource:
+        api_version: core/v2
+        type: Handler
+        name: splunk-metrics-and-events
+      patches:
+        - path: /spec/socket/host
+          op: replace
+          value: "[[host]]"
+        - path: /spec/socket/port
+          op: replace
+          value: "[[port]]"
```

Comment:
Something is going wrong here. The port value is being patched as a string not an int.. and the handler resource fails to install as a result.
I even copied the handler out of the review UI  to a file and used sensuctl create -f to confirm

Reply:
Good find!